2025-01-16 20:55:20,836 - INFO - Finished! Total sfe time: 11.462447881698608 
2025-01-16 20:55:20,836 - DEBUG - 407,server, Parsed server ram usage: 80720
2025-01-16 20:55:20,836 - DEBUG - 407,client, Parsed client ram usage: 80332
2025-01-16 20:55:45,266 - DEBUG - client all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 12 second(s) each.', '2024-12-19 00:53:07,613 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.1:61002', '2024-12-19 00:53:07,616 - tno.mpc.communication.httphandlers - ERROR - Message not received.', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1061, in _wrap_create_connection', '    sock = await aiohappyeyeballs.start_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 103, in start_connection', '    raise first_exception', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 81, in start_connection', '    sock = await _connect_sock(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 165, in _connect_sock', '    await loop.sock_connect(sock, address)', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect', '    return await fut', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb', "    raise OSError(err, f'Connect call failed {address}')", "ConnectionRefusedError: [Errno 111] Connect call failed ('192.168.1.2', 61001)", '', 'The above exception was the direct cause of the following exception:', '', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/tno/mpc/communication/httphandlers.py", line 193, in _send', '    async with self.session.post(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 1355, in __aenter__', '    self._resp: _RetType = await self._coro', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 659, in _request', '    conn = await self._connector.connect(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 557, in connect', '    proto = await self._create_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1002, in _create_connection', '    _, proto = await self._create_direct_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1336, in _create_direct_connection', '    raise last_exc', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1305, in _create_direct_connection', '    transp, proto = await self._wrap_create_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1076, in _wrap_create_connection', '    raise client_error(req.connection_key, exc) from exc', "aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.1.2:61001 ssl:default [Connect call failed ('192.168.1.2', 61001)]", '2024-12-19 00:53:08,687 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-19 00:53:08,719 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-19 00:53:08,809 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-19 00:53:08,858 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', 'TP is  0.001', '2024-12-19 00:53:08,875 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-19 00:53:08,897 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-19 00:53:08,898 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.1:61002 shutdown', 'Total bytes received: 1930', 'Total messages received: 5', '2024-12-19 00:53:08,903 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.2:61001 shutdown', 'Total bytes sent: 766', 'Total messages sent: 5', '2024-12-19 00:53:08,904 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 766', 'Total messages sent: 5', 'Total bytes received: 1930', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 1 407"', '\tUser time (seconds): 6.79', '\tSystem time (seconds): 0.75', '\tPercent of CPU this job got: 87%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.67', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80808', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 16078', '\tVoluntary context switches: 65', '\tInvoluntary context switches: 238', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_rpi_2025-01-16_20-55-09.csv']
2025-01-16 20:55:45,267 - DEBUG - server all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 12 second(s) each.', '2024-12-19 00:53:44,958 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.2:61001', '2024-12-19 00:53:45,970 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-19 00:53:46,082 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-19 00:53:46,110 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-19 00:53:46,160 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', 'TP is  0.001', '2024-12-19 00:53:46,206 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-19 00:53:46,211 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-19 00:53:46,213 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.2:61001 shutdown', 'Total bytes received: 766', 'Total messages received: 5', '2024-12-19 00:53:46,216 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.1:61002 shutdown', 'Total bytes sent: 1930', 'Total messages sent: 5', '2024-12-19 00:53:46,217 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 1930', 'Total messages sent: 5', 'Total bytes received: 766', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 0 407"', '\tUser time (seconds): 6.75', '\tSystem time (seconds): 0.73', '\tPercent of CPU this job got: 87%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.60', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80080', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 15952', '\tVoluntary context switches: 66', '\tInvoluntary context switches: 213', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_rpi_2025-01-16_20-55-09.csv']
2025-01-16 20:55:46,729 - DEBUG - All values from powertop for client: 1       497 mW  
26     1.52 mW  
47      508 uW  
Name: 7, dtype: object
2025-01-16 20:55:46,729 - DEBUG - Energy client: 3.74770028
2025-01-16 20:55:48,045 - DEBUG - All values from powertop for server: 1       507 mW  
13     6.57 mW  
28     1.17 mW  
Name: 7, dtype: object
2025-01-16 20:55:48,045 - DEBUG - Energy server: 3.7884864
